src/vs/workbench/contrib/debug/browser/debugActionViewItems.ts

```diff
@@ -75,6 +75,7 @@ export class StartDebugActionViewItem extends BaseActionViewItem {
 		const keybindingLabel = keybinding ? ` (${keybinding})` : '';
 		this.start.title = this.action.label + keybindingLabel;
 		this.start.setAttribute('role', 'button');
+		this.start.ariaLabel = nls.localize('startDebug', "Start Debugging");
```

Comment:
Shouldn't this be the same as `title`? Otherwise we have an extra string to translate (and no keybinding):

![image](https://github.com/microsoft/vscode/assets/2193314/f7a17760-2b6c-43ae-bcac-19ea07b4d196)

Reply:
Perhaps we should follow suit here like you suggested but consider an `accessibility.verbosity` setting which excludes the keybinding part across the board?